src/Files.App/Filesystem/StorageEnumerators/Win32StorageEnumerator.cs

```diff
@@ -171,115 +166,137 @@ public static ListedItem GetAlternateStream((string Name, long Size) ads, Listed
 		}
 
 		public static async Task<ListedItem> GetFolder(
-			Backend.Helpers.NativeFindStorageItemHelper.WIN32_FIND_DATA findData,
+			NativeFindStorageItemHelper.WIN32_FIND_DATA findData,
 			string pathRoot,
+			bool isGitRepo,
 			CancellationToken cancellationToken
 		)
 		{
 			if (cancellationToken.IsCancellationRequested)
-			{
 				return null;
-			}
 
 			DateTime itemModifiedDate;
 			DateTime itemCreatedDate;
+
+			// Convert FileTime To SystemTime (Universal Time To Local Time)
 			try
 			{
-				FileTimeToSystemTime(ref findData.ftLastWriteTime, out Backend.Helpers.NativeFindStorageItemHelper.SYSTEMTIME systemModifiedTimeOutput);
+				FileTimeToSystemTime(ref findData.ftLastWriteTime, out NativeFindStorageItemHelper.SYSTEMTIME systemModifiedTimeOutput);
 				itemModifiedDate = systemModifiedTimeOutput.ToDateTime();
 
-				FileTimeToSystemTime(ref findData.ftCreationTime, out Backend.Helpers.NativeFindStorageItemHelper.SYSTEMTIME systemCreatedTimeOutput);
+				FileTimeToSystemTime(ref findData.ftCreationTime, out NativeFindStorageItemHelper.SYSTEMTIME systemCreatedTimeOutput);
 				itemCreatedDate = systemCreatedTimeOutput.ToDateTime();
 			}
 			catch (ArgumentException)
 			{
-				// Invalid date means invalid findData, do not add to list
+				// Invalid WIN32_FIND_DATA data
 				return null;
 			}
+
 			var itemPath = Path.Combine(pathRoot, findData.cFileName);
+
 			string itemName = await fileListCache.ReadFileDisplayNameFromCache(itemPath, cancellationToken);
 			if (string.IsNullOrEmpty(itemName))
-			{
 				itemName = findData.cFileName;
-			}
-			bool isHidden = (((FileAttributes)findData.dwFileAttributes & FileAttributes.Hidden) == FileAttributes.Hidden);
```

Comment:
Let's please keep things in scope to make it easier to review.

Reply:
Maybe it's just me trying to reduce my workload but on a serious note, it can be confusing when there are unrelated changes.